### PR TITLE
add jars for MongoDB 7

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: lib/jars/mongo/mongodb-driver-core-4.11.1.jar
Comment (MrCreosote) [inline lib/jars/mongo/mongodb-driver-core-4.11.1.jar:1]: It looks like the core driver has a required runtime dependency on [bson-record-codec](https://mvnrepository.com/artifact/org.mongodb/bson-record-codec) : https://mvnrepository.com/artifact/org.mongodb/mongodb-driver-core/4.11.1

Comment (MrCreosote) [reply lib/jars/mongo/mongodb-driver-core-4.11.1.jar:1]: And yes we need to move to gradle

Comment (Xiangs18) [reply lib/jars/mongo/mongodb-driver-core-4.11.1.jar:1]: Added. I will work on grade once I can run pass all tests.

Comment (MrCreosote) [reply lib/jars/mongo/mongodb-driver-core-4.11.1.jar:1]: Yeah, don't start on gradle yet, mongo upgrades come first. I've started work on gradlization and so far it's a bit of a beast, lots on strange test failures that work fine with ant

**Review by MrCreosote — COMMENTED** [review]
This LGTM but historically we've had @kkellerlbl approve all jars changes. Try pinging him in slack and see what he says

**Review by kkellerlbl — APPROVED** [review]
If it LGTGavin it LGTM

**Pull request merged**